### PR TITLE
DOCS-1098: Add link to modular resource examples in relevant resource pages

### PR DESCRIPTION
**Pull request by sguequierre**
First five examples are now covered-- last three are a little ambigious and unsure whether/where to add (tensorflow-lite for what the model name is and how to add it to docs, last two are old custom base tutorials)

### CODE REVIEW

File: docs/components/motor/_index.md [hunk 1]
```diff
@@ -41,6 +41,13 @@ Model | Description <a name="model-table"></a>
 [`roboclaw`](./roboclaw/) | [Standard brushed DC motor](https://en.wikipedia.org/wiki/DC_motor) driven by [Basicmicro's](https://www.basicmicro.com/) [RoboClaw](https://www.basicmicro.com/RoboClaw-2x30A-Motor-Controller_p_9.html) motor controller
 [`fake`](./fake/) | Used to test code without hardware
 
+Viam also provides the following motor models as [modular resources](/extend/modular-resources/):
+
+ Model | Description
+ ----- | -----------
+ [`viam:odrive:canbus`](/extend/modular-resources/examples/odrive/) | An [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver with CANbus communication
+ [`viam:odrive:serial`](/extend/modular-resources/examples/odrive/) | An [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver with serial communication
+
```
Comment (andf-viam) [inline docs/components/motor/_index.md:50]: Let's add one sentence after the modules with a bit of next steps, if you agree. Same for other instances in this PR.

> These modules can be [added to your robot from the Viam registry](https://docs.viam.com/extend/modular-resources/configure/#add-a-module-from-the-viam-registry)

**Review by andf-viam — APPROVED** [review]
LGTM! Just one optional suggestion. Thanks much!

File: docs/components/motor/_index.md (hunk 1 above)
Comment (sguequierre) [inline docs/components/motor/_index.md:50]:
```suggestion

These modules can be [added to your robot from the Viam registry](https://docs.viam.com/extend/modular-resources/configure/#add-a-module-from-the-viam-registry)
```

Comment (sguequierre) [reply docs/components/motor/_index.md:50]: I'm good with that, seems like a good chance to advertise the registry! Thanks andrew!

File: docs/components/sensor/_index.md [hunk 2]
```diff
@@ -48,6 +48,12 @@ This allows you to have the same access and control of the sensor through Viam a
 
 For an example of creating a custom component, see a [WiFi strength sensor built with the Viam Go SDK](https://github.com/viam-labs/wifi-sensor/blob/main/linuxwifi/linuxwifi.go) or [custom resource types implemented with the Viam Python SDK](https://github.com/viamrobotics/viam-python-sdk/tree/main/examples/).
 
+Viam also provides the following sensor models as [modular resources](/extend/modular-resources/):
+
+ Model | Description
+ ----- | -----------
+ [`viam:visual_odometry:opencv_orb`](/extend/modular-resources/examples/odrive/) | A resource which uses monocular [visual odometry](https://en.wikipedia.org/wiki/Visual_odometry) to enable any [calibrated cameras](/components/camera/calibrate/) to function as a movement sensor
+
```
Comment (sguequierre) [inline docs/components/sensor/_index.md:56]:
```suggestion

This module can be [added to your robot from the Viam registry](https://docs.viam.com/extend/modular-resources/configure/#add-a-module-from-the-viam-registry).
```

File: docs/components/motor/_index.md [hunk 3]
```diff
@@ -41,6 +41,14 @@ Model | Description <a name="model-table"></a>
...
 [`fake`](./fake/) | Used to test code without hardware
 
+Viam also provides the following motor models as [modular resources](/extend/modular-resources/):
+
+ Model | Description
+ ----- | -----------
+ [`viam:odrive:canbus`](/extend/modular-resources/examples/odrive/) | An [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver with CANbus communication
+ [`viam:odrive:serial`](/extend/modular-resources/examples/odrive/) | An [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver with serial communication
+
+These modules can be [added to your robot from the Viam registry](https://docs.viam.com/extend/modular-resources/configure/#add-a-module-from-the-viam-registry)
```
Comment (sguequierre) [inline docs/components/motor/_index.md:51]:
```suggestion
These modules can be [added to your robot from the Viam registry](https://docs.viam.com/extend/modular-resources/configure/#add-a-module-from-the-viam-registry)

```

File: docs/components/sensor/_index.md [hunk 4]
```diff
@@ -48,6 +48,13 @@ This allows you to have the same access and control of the sensor through Viam a
 
 For an example of creating a custom component, see a [WiFi strength sensor built with the Viam Go SDK](https://github.com/viam-labs/wifi-sensor/blob/main/linuxwifi/linuxwifi.go) or [custom resource types implemented with the Viam Python SDK](https://github.com/viamrobotics/viam-python-sdk/tree/main/examples/).
 
+Viam also provides the following sensor model as a [modular resource](/extend/modular-resources/):
+
+ Model | Description
+ ----- | -----------
+ [`viam:visual_odometry:opencv_orb`](/extend/modular-resources/examples/odrive/) | A resource which uses monocular [visual odometry](https://en.wikipedia.org/wiki/Visual_odometry) to enable any [calibrated cameras](/components/camera/calibrate/) to function as a movement sensor
+
+This module can be [added to your robot from the Viam registry](https://docs.viam.com/extend/modular-resources/configure/#add-a-module-from-the-viam-registry).
```
Comment (sguequierre) [inline docs/components/sensor/_index.md:57]:
```suggestion
This module can be [added to your robot from the Viam registry](https://docs.viam.com/extend/modular-resources/configure/#add-a-module-from-the-viam-registry).

```

File: docs/components/sensor/_index.md [hunk 5]
```diff
@@ -48,6 +48,14 @@ This allows you to have the same access and control of the sensor through Viam a
...
```
Comment (sguequierre) [inline docs/components/sensor/_index.md:57]:
```suggestion
This module can be [added to your robot from the Viam registry](/extend/modular-resources/configure/#add-a-module-from-the-viam-registry).
```

File: docs/components/sensor/_index.md [hunk 6]
```diff
@@ -48,6 +48,14 @@ This allows you to have the same access and control of the sensor through Viam a
 
 For an example of creating a custom component, see a [WiFi strength sensor built with the Viam Go SDK](https://github.com/viam-labs/wifi-sensor/blob/main/linuxwifi/linuxwifi.go) or [custom resource types implemented with the Viam Python SDK](https://github.com/viamrobotics/viam-python-sdk/tree/main/examples/).
 
+Viam also provides the following sensor model as a [modular resource](/extend/modular-resources/):
+
+ Model | Description
+ ----- | -----------
+ [`viam:visual_odometry:opencv_orb`](/extend/modular-resources/examples/odrive/) | A resource which uses monocular [visual odometry](https://en.wikipedia.org/wiki/Visual_odometry) to enable any [calibrated cameras](/components/camera/calibrate/) to function as a movement sensor
+
+This module can be [added to your robot from the Viam registry](/extend/modular-resources/configure/#add-a-module-from-the-viam-registry).
```
Comment (sguequierre) [inline docs/components/sensor/_index.md:57]:
```suggestion
This module can be [added to your robot from the Viam registry.](/extend/modular-resources/configure/#add-a-module-from-the-viam-registry).
```

File: docs/components/motor/_index.md [hunk 7]
```diff
@@ -41,6 +41,15 @@ Model | Description <a name="model-table"></a>
...
 [`fake`](./fake/) | Used to test code without hardware
 
+Viam also provides the following motor models as [modular resources](/extend/modular-resources/):
+
+ Model | Description
+ ----- | -----------
+ [`viam:odrive:canbus`](/extend/modular-resources/examples/odrive/) | An [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver with CANbus communication
+ [`viam:odrive:serial`](/extend/modular-resources/examples/odrive/) | An [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver with serial communication
+
+These modules can be [added to your robot from the Viam registry](/extend/modular-resources/configure/#add-a-module-from-the-viam-registry)
```
Comment (sguequierre) [inline docs/components/motor/_index.md:51]:
```suggestion
These modules can be [added to your robot from the Viam registry](/extend/modular-resources/configure/#add-a-module-from-the-viam-registry).

```

File: docs/components/sensor/_index.md [hunk 8]
```diff
@@ -48,6 +48,14 @@ This allows you to have the same access and control of the sensor through Viam a
 
 For an example of creating a custom component, see a [WiFi strength sensor built with the Viam Go SDK](https://github.com/viam-labs/wifi-sensor/blob/main/linuxwifi/linuxwifi.go) or [custom resource types implemented with the Viam Python SDK](https://github.com/viamrobotics/viam-python-sdk/tree/main/examples/).
 
+Viam also provides the following sensor model as a [modular resource](/extend/modular-resources/):
+
+ Model | Description
+ ----- | -----------
+ [`viam:visual_odometry:opencv_orb`](/extend/modular-resources/examples/odrive/) | A resource which uses monocular [visual odometry](https://en.wikipedia.org/wiki/Visual_odometry) to enable any [calibrated cameras](/components/camera/calibrate/) to function as a movement sensor
+
+This module can be [added to your robot from the Viam registry.](/extend/modular-resources/configure/#add-a-module-from-the-viam-registry).
```
Comment (sguequierre) [inline docs/components/sensor/_index.md:57]:
```suggestion
This module can be [added to your robot from the Viam registry](/extend/modular-resources/configure/#add-a-module-from-the-viam-registry).
```

File: docs/components/motor/_index.md [hunk 9]
```diff
@@ -41,6 +41,16 @@ Model | Description <a name="model-table"></a>
...
 
+Viam also provides the following motor models as [modular resources](/extend/modular-resources/):
+
+ Model | Description
+ ----- | -----------
+ [`viam:odrive:canbus`](/extend/modular-resources/examples/odrive/) | An [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver with CANbus communication
+ [`viam:odrive:serial`](/extend/modular-resources/examples/odrive/) | An [ODrive S1](https://odriverobotics.com/shop/odrive-s1) motor driver with serial communication
+
+These modules can be [added to your robot from the Viam registry](/extend/modular-resources/configure/#add-a-module-from-the-viam-registry).
+
```
Comment (sguequierre) [inline docs/components/motor/_index.md:52]:
```suggestion
```

**Pull request merged**